### PR TITLE
Companies UI PR

### PR DESCRIPTION
**Pull request by chelseyywang**
- Companies Admin View
- added Link from "Update" on each Company Card to Update Company Form with corresponding company's original values pre-filled on form
- fixed queries in Company Update Form
- Successfully updates Company as seen below

Screen Shot 2020-10-15 at 2 32 01 PM
Screen Shot 2020-10-15 at 2 33 39 PM
Screen Shot 2020-10-15 at 2 34 17 PM

TODO: 

- Add redirect/success page after clicking submit on form
- Add functionality for search bar
- Add uploading image functionality for each company
- Fix lag for "positions" checkboxes
- Fix rearranging of Company Cards; if a Card gets updated it is pushed to the bottom of the list

### CODE REVIEW

File: client/src/components/Companies.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	/*
+	componentDidMount() {
+		this.props.onRef(this); 
+	}
+	
+	componentWillUnmount() {
+    	this.props.onRef(null);
+  	}
+  	*/
+
```
Comment (nikhitashanker) [inline client/src/components/Companies.js:38]: Can remove this commented part.

File: client/src/components/CompaniesAdmin.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+	/*
+	componentDidMount() {
+		this.props.onRef(this); 
+	}
+	
+	componentWillUnmount() {
+    	this.props.onRef(null);
+  	}
+  	*/
```
Comment (nikhitashanker) [inline client/src/components/CompaniesAdmin.js:39]: Ditto.

File: client/src/components/CompanyCard.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+          </Typography>)
+    let majorsDisplay = (<Typography>
+            Majors: {this.mapIDToName(majors, allMajors)}
+          </Typography>)
+
+    return (
+        <Card className={classes.card}>
+        <CardMedia
+          className={classes.cardMedia}
+            image="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%22288%22%20height%3D%22225%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20288%20225%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_164edaf95ee%20text%20%7B%20fill%3A%23eceeef%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A14pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_164edaf95ee%22%3E%3Crect%20width%3D%22288%22%20height%3D%22225%22%20fill%3D%22%2355595c%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2296.32500076293945%22%20y%3D%22118.8%22%3EThumbnail%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E" // eslint-disable-line max-len
```
Comment (nikhitashanker) [inline client/src/components/CompanyCard.js:114]: Is this file something you have locally?

File: client/src/components/CompaniesAdmin.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+		              SWE Admin Access
+		            </Typography>
+		          </div>
+		        </div>
+		        <div>
+        		<Fab color="primary" aria-label="add" href="/companiesform">
+			        <AddIcon />
+			     </Fab>
+	           </div>
+		        <AdminCompaniesCardList/>
```
Comment (nikhitashanker) [inline client/src/components/CompaniesAdmin.js:72]: Format looks a bit off here.

File: client/src/components/CompaniesAdmin.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import AdminCompanyCard from './AdminCompanyCard.js';
+import AdminCompaniesCardList from './AdminCompaniesCardList.js';
+import CompaniesStyles from '../styles/Companies.js';
+import Fab from '@material-ui/core/Fab';
+import AddIcon from '@material-ui/icons/Add';
+
+
+const cards = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
```
Comment (nikhitashanker) [inline client/src/components/CompaniesAdmin.js:24]: I think you can remove this.

File: client/src/components/CompanyCard.js (hunk 3 above)
Comment (chelseyywang) [reply client/src/components/CompanyCard.js:114]: no

**Review by nikhitashanker — APPROVED** [review]
Please add the issues and link issues to milestones, so we can practice the workflow!

**Review by christinatong01 — APPROVED** [review]

**Pull request merged**